src/mds/CInode.h

```diff
@@ -87,6 +87,7 @@ struct CInodeCommitOperations {
   inode_backtrace_t bt;
   version_t version;
   CInode *in;
+  std::string_view symlink;
```

Comment:
How about moving this member to `CInodeCommitOperation` class instead. And initialize it in `_store_backtrace()` instead, just like the `_layout`.  Then we can also be sure that for each `CInode` the `symlink` will be initialized only once.

Reply:
You mean to say, with this approach, whenever the parents becomes dirty, we would just update symlink xattr even though it's not required ?

Reply:
Ok, I get it. Done.